module-4/backend/week_one.md

```diff
@@ -5,15 +5,34 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+For me, I try to take the intermission work as initial exposure to whatever concepts we'll be learning during the mod. I try to digest the material as muscle memory and syntax exposure.
 2. What are some tools to help debug JavaScript code?
+DevTools, debugger, pryjs, console.log
 3. What are some tools you need in order to unit test your JavaScript?
+mocha
 4. What is the syntax for invoking a function?
```

Comment:
So these are ways to 'define' a function to 'invoke' or 'call' a function you write the function name with parenthesis on the end `functionName()`